### PR TITLE
fix(docs): add  protoc-gen-openapiv2 to tutorial

### PR DESCRIPTION
**Pull request by Fahmadi**
#### Brief description of what is fixed or changed
To use `grpc-gateway` we should install `protoc-gen-openapiv2` as mentioned in the [README.md](https://github.com/grpc-ecosystem/grpc-gateway)

but this line is missing [in the tutorials page document](https://grpc-ecosystem.github.io/grpc-gateway/docs/tutorials/introduction/)

I had difficulties installing the `openapiv2` required for `grpc-gateway`

### CODE REVIEW

**Comment by johanbrandhorst** [comment]
Hi Fatemeh, thank you for your PR! I'm not sure this change is actually correct - nothing in the tutorial uses the OpenAPIv2 generator as far as I can tell. Could you explain why you think this change is necessary? You say you were unable to run the tutorial, but the tutorial doesn't use the openapiv2 generator anywhere. Were you switching between the tutorial and the README perhaps? I can understand the confusion, and maybe we should add a section about how to use the openapiv2 generator too, or at least link to somewhere where the user could learn more about it. Regardless, I don't think this change, in isolation, is right. What do you think?

**Comment by Fahmadi** [comment]
Hi @johanbrandhorst. Thanks for your response. I made a mistake. my difficulties were not related to openApiV2.

In `grpc-gateway` tutorial, it was written that in order to get `grpc-gateway` you should use the `go get` command, and as you know this can not install any binary programs because [starting in Go 1.17, installing executables with go get is deprecated.
](https://go.dev/doc/go-get-install-deprecation)

so when I ran the program I got this error: 

```
Failure: plugin grpc-gateway: could not find protoc plugin for name grpc-gateway
```

because `grpc-gateway` did not exist in directory `~/go/bin`. so I think these three commands in the tutorial should be changed to the `get install` command as mentioned in [README.md](https://github.com/grpc-ecosystem/grpc-gateway)

File: docs/docs/tutorials/introduction.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ We will be using a Go gRPC server in the examples, so please install Go first fr
 After installing Go, use `go get` to download the following packages:
```
Comment (johanbrandhorst) [inline docs/docs/tutorials/introduction.md:30]: Lets change this as well
```suggestion
After installing Go, use `go install` to download and build the following binaries:
```

File: docs/docs/tutorials/introduction.md [hunk 2]
```diff
@@ -30,9 +30,9 @@ We will be using a Go gRPC server in the examples, so please install Go first fr
...
 
 ```sh
-$ go get github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
-$ go get google.golang.org/protobuf/cmd/protoc-gen-go
-$ go get google.golang.org/grpc/cmd/protoc-gen-go-grpc
+$ go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
+$ go install google.golang.org/protobuf/cmd/protoc-gen-go
+$ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
```
Comment (johanbrandhorst) [inline docs/docs/tutorials/introduction.md:35]: I believe this requires a version too
```suggestion
$ go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@latest
$ go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
$ go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
I see, great point! Some minor update suggestions.

**Comment by Fahmadi** [comment]
I resolved the changes, so please approve workflows

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Thank you for your contribution!

**Pull request merged**